sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClientBuilder.java

```diff
@@ -29,7 +29,7 @@
 @ServiceClientBuilder(serviceClients = {TableServiceClient.class, TableServiceAsyncClient.class})
 public class TableServiceClientBuilder {
     private final ClientLogger logger = new ClientLogger(TableServiceClientBuilder.class);
-    private final SerializerAdapter serializerAdapter = new TablesJacksonSerializer();
+    private final SerializerAdapter serializerAdapter = new JacksonAdapter();
```

Comment:
Does that mean we can just delete the other serializer now?

Reply:
No, we still need it for the TableClient, but the TableServiceClient doesn't need any customization so it can just use the default JacksonAdapter.

Reply:
Sweet thanks!

Reply:
Fixed